postgresql.json

```
@@ -0,0 +1,34 @@
+{
+    "PostgreSQL":{
+        "containers":{
+            "postgresql":{
+                "image":"postgres:10.6",
```

Use "tag" for specifying image version.


```suggestion
                "image": "postgres",
                "tag": "10.6",
```